packages/eas-cli/src/update/utils.ts

```diff
@@ -181,6 +193,20 @@ export function formatUpdateTitle(update: UpdateFragment): string {
   )} by ${actorName}, runtimeVersion: ${runtimeVersion}] ${message}`;
 }
 
+export function getUpdateGroupJson(updateGroups: UpdateFragment[]): UpdateJson[] {
```

Comment:
This naming is a bit confusing, this object is not JSON, this an object that you will later serialize into JSON.

The easiest solution is to pass a list of fragments directly `printJsonOnlyOutput`, but if you want to limit that information, then maybe `getUpdateGroupInfo` or `getUpdateGroupForJsonOnlyMode`

Reply:
Sound good, I wasn't really happy with the naming either, but had to put something. Maybe `getUpdateGroupInfo` is slightly too ambiguous compared to `getUpdateGroupDescription` (list output). How about `getUpdateGroupJsonOuput` as opposed to `JsonOnlyMode`? @wkozyra95